### PR TITLE
v1.0.0 Release Notes

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**
Signed-off-by: Pascal Brand <pascal.brand@st.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+
+OP-TEE is now maintained by Linaro. Contributors do not need to
+sign a CLA anymore, but must follow the rules of the [DCO][DCO]
+(Developer Certificate of Origin) instead.
+
+
+## New features
+
+* Add hardware support for
+  * Texas Instruments DRA7xx, ARMv7 (plat-ti)
```
Comment (jforissier) [inline CHANGELOG.md]: One line since there is only one platform

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,61 @@
...
+
+* GlobalPlatform™ TEE Internal Core API Specification v1.1 compliant.
+  It includes ECC algorithms support.
```
Comment (jforissier) [inline CHANGELOG.md]: "Add support for GP ... v1.1, including ECC algorithms."

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,61 @@
...
+
+* Secure Storage: Files stored by the REE are now encrypted. Operations
+  are made atomic in order to prevent errors in the middle of file
+  storage operations. [Slides][LCStorage] describing the Secure Storage
+  have been presented during the Linaro Connect SFO15.
```
Comment (jforissier) [inline CHANGELOG.md]: s/prevent errors/prevent inconsistencies in case of errors/
s/during the/at/

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,61 @@
...
+
+| Type | Meaning |
+| ---- | ------- |
+| Standard tests | The [optee_test][optee_test] project. |
+| Extended tests | optee_test with tests from the GlobalPlatform™ TEE Initial Configuration Test Suite v1.1.0.4. |
+| Hello world test | Plain hello world Trusted Application such as [this][hello_world]. |
+
+*	ARM Juno Board (vexpress-juno), standard tests.
+*	Foundation Models (vexpress-fvp), standard tests.
+*	HiKey (hikey), standard tests.
```
Comment (jforissier) [inline CHANGELOG.md]: I will run extended tests also

**Comment by pascal-brand-st-dev** [comment]
Addressed Jerome's comments

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Pull request merged**